core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/environment/network/EnvironmentBaseNetworkConverter.java

```diff
@@ -30,6 +32,22 @@ public BaseNetwork convert(EnvironmentNetworkV4Request source, Environment envir
         result.setEnvironment(environment);
         result.setWorkspace(environment.getWorkspace());
         result.setSubnetIds(source.getSubnetIds());
+        result.setSubnetCidrs(source.getSubnetCidrs());
+        result.setNetworkCidr(source.getNetworkCidr());
+        result.setRegistrationType(RegistrationType.EXISTING);
+        return result;
+    }
+
+    @Override
+    public BaseNetwork convertNewNetwork(EnvironmentNetworkV4Request source, Environment environment, CreatedCloudNetwork createdCloudNetwork) {
+        BaseNetwork result = createProviderSpecificNetwork(source, createdCloudNetwork);
+        result.setName(environment.getName());
```

Comment:
7 lines here are duplicated code, can you move these to a private method? e.g. `setBasicProperties`

Reply:
```

    @Override
    public BaseNetwork convert(EnvironmentNetworkV4Request source, Environment environment) {
        BaseNetwork result = createProviderSpecificNetwork(source);
        setBasicProperties(source, environment, result, RegistrationType.EXISTING);
        return result;
    }

    @Override
    public BaseNetwork convertNewNetwork(EnvironmentNetworkV4Request source, Environment environment, CreatedCloudNetwork createdCloudNetwork) {
        BaseNetwork result = createProviderSpecificNetwork(source, createdCloudNetwork);
        setBasicProperties(source, environment, result, RegistrationType.CREATE_NEW);
        return result;
    }

    private void setBasicProperties(EnvironmentNetworkV4Request source, Environment environment, BaseNetwork result, RegistrationType existing) {
        result.setName(environment.getName());
        result.setEnvironment(environment);
        result.setWorkspace(environment.getWorkspace());
        result.setSubnetIds(source.getSubnetIds());
        result.setSubnetCidrs(source.getSubnetCidrs());
        result.setNetworkCidr(source.getNetworkCidr());
        result.setRegistrationType(existing);
    }

```